clang/test/dpct/implicit_include.cu

```diff
@@ -0,0 +1,10 @@
+// RUN: dpct -out-root %T/implicit_include %s --cuda-include-path="%cuda-path/include"
+// RUN: FileCheck --input-file %T/implicit_include/implicit_include.dp.cpp --match-full-lines %s
+
+// CHECK: #include <cmath>
```

Comment:
if "#include <cmath>" already there,  will the PR insert another "#include <cmath>",  if yes, we need further refine.

Reply:
I checked our code, we already supported this case.
I added a new lit case to test it.